Installer/create_install_script.py

```diff
@@ -1,42 +1,70 @@
 #!python3
 
-# Run this on a Mac or Linux machine to create/update 'install_uartremote.py'
-# Copy the contents of install_uartremote.py into an empty SPIKE Prime project
-# And run to install
+# Run this on a Mac or Linux machine to create/update 'install_mpy_robot_tools.py'
+# Copy the contents of install_mpy_robot_tools.py into an empty SPIKE/51515 project
+# And run to install. The transfer and program run will take extra time.
+# to restore to OEM, allow LEGO app to update your hub, tools will be rempved.
+# Optionally if you clone https://github.com/antonvh/UartRemote and uartremote.py
+# will be installed with mpy_robot_tools
+#
+# Open the console/Debug and watch for notice to unplug USB BEFORE you upload/run
 
 import binascii, mpy_cross, time
 import hashlib
 import os
 from functools import partial
 
 LIB = '../mpy_robot_tools/'
+LIB_UART = '../../LMS-uart-esp/Libraries/UartRemote/MicroPython/uartremote.py'
```

Comment:
Please remove this. I would like to do this with a git submodule. That's more reliable.